ocaml/gencert/lib.ml

```diff
@@ -33,6 +36,14 @@ let get_management_ip_addr ~dbg =
       Some (List.hd addrs)
   with _ -> None
 
+let call_generate_ssl_cert ~args =
+  let home_env = match Sys.getenv_opt "HOME" with
+  | None -> D.warn "environment variable 'HOME' is unavailable"; ""
+  | Some h -> "HOME" ^ h
+  in
+  let env = [| "PATH=" ^ (String.concat ":" Forkhelpers.default_path); home_env |] in
+  Forkhelpers.execute_command_get_output generate_ssl_cert ~env args
```

Comment:
Does it make sense to use Xapi_globs or Xapi_consts here to remove the duplicated definition?